src/parcsr_ls/par_mgr_interp.c

```diff
@@ -731,171 +733,159 @@ hypre_ExtendWtoPHost(HYPRE_Int      P_nr_of_rows,
 
 HYPRE_Int
 hypre_MGRBuildPHost( hypre_ParCSRMatrix   *A,
+                     hypre_ParCSRMatrix   *A_FF,
```

Comment:
It would be good to add some comments about the return values of this function and what it means when they are NULL or not NULL, etc.

Reply:
Yes, I will add some comments!

Reply:
Done in [a746eae](https://github.com/hypre-space/hypre/pull/1121/commits/a746eaec1828562c240f639a1d44334285df301b)